### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by goern**
see https://github.com/operate-first/apps/pull/162 for discussion

### CODE REVIEW

**Review by tumido — APPROVED** [review]

File: OWNERS [hunk 1]
```diff
@@ -1,18 +1,14 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - anishasthana
-  - hemajv
   - HumairAK
   - tumido
```
Comment (anishasthana) [inline OWNERS:5]: 4n4nd should also be an approver

File: OWNERS [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
-  - martinpovolny
-  - 4n4nd
-  - durandom
 
 reviewers:
   - anishasthana
   - hemajv
-  - HumairAK
```
Comment (anishasthana) [inline OWNERS:15]: @tumido Weren't we planning on keeping a small overlap between approvers and reviewers?

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 2 above)
Comment (tumido) [reply OWNERS:15]: Ah, you're right. :+1:  Especially with the https://github.com/thoth-station/thoth-application/pull/805 settings change, it make sense. @goern can you please change this back?

**Review by tumido — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 2 above)
Comment (goern) [reply OWNERS:15]: well, not being in the reviewers list just prevents someone from being picked and assigned as a reviewer. see https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process 

or do you want Humair to be a reviewer so he can add the lgtm label?!

Comment (tumido) [reply OWNERS:15]: I think we both want to be reviewers as well. I mean, even if I'm an approver I might want to `lgtm` something while still wanting a second pair of eyes on that PR before merging.. Do I make sense or is that totally wrong expectation I have?

**Comment by HumairAK** [comment]
Closing in exchange for #74

**Pull request closed without merging**